README.md

```diff
@@ -80,9 +81,11 @@ Notes:
 # Software Installation
 
 ## A Special Note On Minimizing Trust
-As is the nature of pre-packaged software downloads, downloading and using the prepared SeedSigner release images means implicitly placing trust in the individual preparing those images; in our project the release images are prepared and signed by the eponymous creator of the project, SeedSigner "the person". That individual is additionally the only person in possession of the PGP keys that are used to sign the release images.
+As is the nature of pre-packaged software downloads, downloading and using the prepared SeedSigner release images means implicitly placing trust in the people preparing those images; in our project the released images are prepared and signed by the eponymous creator of the project, SeedSigner "the person". That individual is additionally the only person in possession of the PGP keys that are used to sign the release images.
 
-However, one of the many advantages of the open source software model is that the need for this kind of trust can be negated by our users' ability to (1) review the project's source code and (2) assemble the operating image necessary to use the software themselves. From our project's inception, instructions to build a SeedSigner operating image (using precisely the same process that is used to create the prepared release images) have been made available. We have put a lot of thought and work into making these instructions easy to understand and follow, even for less technical users. These instructions can be found [here](docs/manual_installation.md).
+As of release 0.7.0 the images distributed via GitHub are reproducible. This means you and others can verify the released images are byte for byte the same when built from source. If you have the ability to read and understand source code; then you can contribute to this project by building from source and sharing the hash of the final images.
```

Comment:
"As of release 0.7.0 the images" -> "Starting with v0.7.0, the images..." (minor edit suggestion, add comma)

"are byte for byte the same" -> "are byte-for-byte the same" (non-critical edit)

"If you have the ability to read and understand source code; then you can contribute..." -> "You can contribute to this project..." (just the act of reproducing the build is itself a contribution via verification. No code reading ability required)